src/applications/claims-status/components/ClaimsListItem.jsx

```diff
@@ -102,13 +102,7 @@ export default function ClaimsListItem({ claim }) {
       </div>
       {showAlert && (
         <va-alert status="info" slim>
-          <span className="vads-u-font-weight--bold">
-            We requested more information from you:
-          </span>{' '}
-          Check the claim details to learn more.
-          <div className="vads-u-margin-top--2">
-            This message will go away when we finish reviewing your response.
-          </div>
+          Open request: Check claim to see if action is needed from you or VA.
```

Comment:
Is this change supposed to be here?

Reply:
@jerekshoe good question - I'm taking over this PR from another engineer, I will investigate!

Reply:
Flagging this so it doesn't get accidentally merged or approved until resolved.